### PR TITLE
add transparency log search ui provided by chainguard team

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

# Changes

Add new annotation that provides a UI for searching transparency log

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [x] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

``` release-note
add transparency log search ui provided by chainguard team
```

/cc @wlynch @priyawadhwa @imjasonh

### CODE REVIEW

**Comment by developer-guy** [comment]
I think we need to explain that this service has no SLOs, so there will be no guarantee that it will always be up and running ☝️

**Comment by imjasonh** [comment]
I would recommend against this. The Rekor Search UI is experimental, unsupported, and reserves the right to go away at any time without warning. By contrast, labels are forever.

If folks want to document how to look up a Rekor entry in the UI based on existing annotations on resources, that seems fine. If the Rekor UI goes away we can update docs easier than we can update labels for all these resources in every cluster.

**Comment by lcarva** [comment]
@developer-guy, based on @imjasonh's [comment](https://github.com/tektoncd/chains/pull/632#issuecomment-1332290844) I think including the Rekor UI URL is not something we want to add to Chains. Would you be interested in changing the approach of this PR to instead add docs?

**Comment by developer-guy** [comment]
> @developer-guy, based on @imjasonh's [comment](https://github.com/tektoncd/chains/pull/632#issuecomment-1332290844) I think including the Rekor UI URL is not something we want to add to Chains. Would you be interested in changing the approach of this PR to instead add docs?

that'd be great @lcarva, I'll replace the PR's content with a documentation update, thanks.

**Comment by developer-guy** [comment]
Kindly ping @imjasonh @lcarva @wlynch, I've updated the documentation and removed the updates within the codebase.

File: docs/sigstore.md [hunk 1]
```diff
@@ -76,3 +76,18 @@ To enable singing with Fulcio, run:
 ```shell
 kubectl patch configmap chains-config -n tekton-chains -p='{"data":{"signers.x509.fulcio.enabled": "true"}}'
 ```
+
+### Better Way Of Navigating in Transparency Log with rekor-search-ui
+
+There is an already annotation in the `TaskRun` or `PipelineRun,` which is `chains.tekton.dev/transparency`, which contains the transparency log entry ID and this can be used to navigate to the transparency log entry over an API, but if you prefer to use a UI, you can use [rekor-search-ui](https://github.com/chainguard-dev/rekor-search-ui), which is a simple UI for searching and viewing transparency log entries.
```
[the remark this answers is not in the document]
Comment (developer-guy) [reply docs/sigstore.md]: Cool, I've updated it accordingly

**Comment by lcarva** [comment]
Hmm. It looks like a random failure on e2e tests:
> curl: (22) The requested URL returned error: 503

Unfortunately, I don't have access to re-run those tests.

**Comment by developer-guy** [comment]
kindly ping @lcarva

**Review by lcarva — APPROVED** [review]
/lgtm

**Pull request merged**